### PR TITLE
Queries Refactor

### PR DESCRIPTION
**Pull request by sparkplug0025**
This PR is preparing the terrain for #243 , forcing the use of queries across the board and removing arbitrary ones.

What it does:
- centralizes query
- no more arbitrary queries, we reuse the ones defined in the `queries.ts`
- removes useFetchedData hook, now we only fetch through queries
- a little refactor in the inductions container page

### CODE REVIEW

**Comment by brandonfancher** [comment]
This is looking really good.

File: packages/webapp/src/pages/members/[id].tsx [hunk 1]
```diff
@@ -2,23 +2,29 @@ import { GetServerSideProps } from "next";
...
 
+/**
+ * We have an issue if the member is not found in the development environment
+ * due to dehydration with JSON not being able to serialize `undefined`:
+ * Error: Error serializing `.dehydratedState.queries[0].state.data`
+ * returned from `getServerSideProps` in "/members/[id]".
+ * Reason: `undefined` cannot be serialized as JSON. Please use `null`
+ * or omit this value.
+ * Let's track this here: https://github.com/tannerlinsley/react-query/issues/1978
+ */
```
Comment (brandonfancher) [inline packages/webapp/src/pages/members/[id].tsx:22]: I noticed this before. While it fails loudly in development mode, in production, it thankfully falls back gracefully to our "member not found" screen.

**Review by brandonfancher — COMMENTED** [review]
The codebase is getting really clean and sexy. 👏

I did run into this error when trying to delete an expired invitation. It doesn't happen when deleting pending transactions though. See if you can reproduce it. I haven't had a chance to dive into it yet.

https://user-images.githubusercontent.com/7911424/121089238-d9f9ce80-c7b4-11eb-8218-689a94bf4fe0.mp4

**Comment by brandonfancher** [comment]
@sparkplug0025 I reviewed all the code. Looks great. I'd like to run a few more tests on this tomorrow morning, though, before we merge it, if that's okay.

**Comment by brandonfancher** [comment]
> I did run into this error when trying to delete an expired invitation. It doesn't happen when deleting pending transactions though. See if you can reproduce it. I haven't had a chance to dive into it yet.

`edenmember12` has an expired one you can test that with. Curious to see if you get the same error.

**Review by brandonfancher — APPROVED** [review]
Approved...pending a look into the delete expired issue I posted about.

**Comment by sparkplug0025** [comment]
> > I did run into this error when trying to delete an expired invitation. It doesn't happen when deleting pending transactions though. See if you can reproduce it. I haven't had a chance to dive into it yet.
> 
> `edenmember12` has an expired one you can test that with. Curious to see if you get the same error.

This error is happening because of that map with hooks inside a function (as opposed to a component)... [Remember that old mind-blowing comment that we agreed to leave as is and review later?](https://github.com/eoscommunity/Eden/pull/85#pullrequestreview-648047133) It seems like it's time to review it! lol

I will merge this one and work on a refactor for these dangling hooks.

**Pull request merged**